config/f17_config_rb46/params.py

```diff
@@ -0,0 +1,11 @@
+configuration = {
+    'motor_gear_ratio': 12.52,
+    'thooth_count': 15,
+    'pitch': 0.033,
+    'wheel_distance': 1.5,
+    'antenna_offset': 0,
+    'work_x': -0.06933333,
+    'work_y': 0.0094166667,
```

Comment:
```suggestion
    'work_x': 0.0,
    'work_y': 0.0,
```
Since we don't know, better to use 0